### ISSUE TITLE
Anti-Cheat Toolkit copyright infringement

### ISSUE THREAD

**Issue by DmitriyYukhanov** [Masmikh/Project-Wir#1]
Hey, Masmikh!

I'm Dmitriy Yukhanov, author of the Anti-Cheat Toolkit plugin for Unity.

Thanks for purchasing Anti-Cheat Toolkit and for using it in your project(s), I hope it goes well!
But unfortunately, it's not allowed to share it anywhere, as it violates Asset Store Terms of Service and EULA (http://unity3d.com/ru/legal/as_terms), APPENDIX 1, sections 5 & 9 and my own copyrights.

And what is more upset, you have to totally vanish it from the repo, from all commits, which means you have to completely remove repo from the GitHub =(

I hope for the understanding and I'll come back in few days and submit DMCA Takedown Notice if my asset will still be here.

I'm sorry and thank you for your understanding!

**Comment by Masmikh** [comment]
It is done good sir And i am really sorry for this. I did this by mistake.
I should check other plugins as well.

**Comment by Masmikh** [comment]
Oh i just read this part "And what is more upset, you have to totally vanish it from the repo, from all commits, which means you have to completely remove repo from the GitHub =("

Ill do it soon, just give me 1 day.

**Comment by DmitriyYukhanov** [comment]
Sure, no worries, thank you for a quick reaction!

**Comment by DmitriyYukhanov** [comment]
Yeah, unfortunately any commit can be accessed so the only way is to remove whole repo =(

**Comment by DmitriyYukhanov** [comment]
Hey, @Masmikh !
Any news on removal of the repository?

**Comment by Masmikh** [comment]
Hello good sir

I am really sorry, i was away for 4 5 days and i am back now. I'll remove it in 12 hours. Thanks for your patient and time your are putting into this, i really appreciate it. I'll inform you when its done.
